addons/jts/src/test/scala/com/github/tminglei/slickpg/PgPostGISSupportSuite.scala

```diff
@@ -27,7 +27,10 @@ class PgPostGISSupportSuite extends FunSuite {
   import MyPostgresProfile.api._
   import PgGeographyTypes._
 
-  val db = Database.forURL(url = utils.dbUrl, driver = "org.postgresql.Driver")
+  override lazy val imageName: String = "postgis/postgis"
+  override lazy val imageTag: String = "11-3.0"
```

Comment:
[postgis/postgis](https://github.com/postgis/docker-postgis) expands on the official docker `postgres` by installing `postgis` and `postgis-scripts`.